src/test/ui/consts/const-eval/ub-ref-ptr.rs

```diff
@@ -1,5 +1,7 @@
 // ignore-tidy-linelength
 // stderr-per-bitwidth
```

Comment:
Can you check which of these files even still need stderr-per-bitwidth? I assume some of them will have bitwidth-independent output now.